src/panels/SkyBrowserPanel/WorldWideTelescope/WorldWideTelescopeView.tsx

```diff
@@ -8,26 +9,69 @@ import { useWindowSize } from '@/windowmanagement/Window/hooks';
 
 import { useWwtProvider } from './WwtProvider/hooks';
 import {
+  useOverlayStatus,
   useUpdateAim,
   useUpdateBorderColor,
   useUpdateBorderRadius,
   useUpdateOpacities,
   useUpdateSelectedImages
 } from './hooks';
+import { InfoOverlayContent } from './InfoOverlayContent';
+
+import styles from './WorldWideTelescope.module.css';
 
 export function WorldWideTelescopeView() {
+  const nBrowsers = useAppSelector((state) => state.skybrowser.browserIds.length);
+  const id = useAppSelector((state) => state.skybrowser.selectedBrowserId);
+
   const [isDragging, setIsDragging] = useState(false);
   const [startDragPosition, setStartDragPosition] = useState({ x: 0, y: 0 });
 
   const { ref } = useWwtProvider();
-
   const { width, height } = useWindowSize();
   const luaApi = useOpenSpaceApi();
-
-  const nBrowsers = useAppSelector((state) => state.skybrowser.browserIds.length);
-  const id = useAppSelector((state) => state.skybrowser.selectedBrowserId);
+  const { visible, type: overlayType } = useOverlayStatus();
+  /**
+   * The `offset` parameter represents the accumulated distance of the drag gesture
+   * in pixels. It is an array where:
+   * - `offset[0]` corresponds to the horizontal distance (x-axis).
+   * - `offset[1]` corresponds to the vertical distance (y-axis).
+   *
+   * This value is updated as the user drags, providing the total displacement
+   * from the starting point of the gesture.
+   */
+  const bindGestures = useGesture(
+    {
+      onDrag: (state) => {
+        state.event.preventDefault();
+        if (!state.wheeling && !state.pinching && state.touches === 1) {
+          handleDrag(state.offset[0], state.offset[1]);
+        }
+      },
+      onDragStart: (state) => {
+        if (state.touches === 1 && !state.wheeling && !state.pinching) {
+          mouseDown(state.offset[0], state.offset[1]);
+        }
+      },
+      onDragEnd: () => mouseUp(),
+      onPinch: (state) => {
+        if (state.touches == 2) {
+          // The first direction determines the "scroll" direction
+          const [direction] = state.direction;
+          scroll(direction < 0 ? 50 : -50);
+        }
+      },
+      onWheel: (state) => {
+        scroll(state.event.deltaY);
+      }
+    },
+    {
+      // To make it differentiate better between dragging and pinching
+      drag: { threshold: 0.1 },
+      eventOptions: { passive: false }
+    }
+  );
```

Comment:
Can this maybe be made a hook with a name like `useSkyBroweserTouchGestures` and moved to the hooks file? Having it here makes the code quite hard to overview, especially with the comment over the function